src/services/BoardApi.js

```diff
@@ -135,6 +135,21 @@ export class BoardApi {
 			})
 	}
 
+	cloneBoard(board) {
+		return axios.post(this.url(`/boards/${board.id}/clone`))
```

Comment:
please use try catch syntax :)

Reply:
Could you give me a hint, how?

Reply:
Hey! https://javascript.info/async-await :)

Reply:
Thanks, now it looks much better! :)